radis/tools/slit.py

```diff
@@ -714,15 +787,27 @@ def get_FWHM(w, I, return_index=False):
     """
     # TODO: Linearly interpolate at the boundary? insignificant for large number of points
 
-    upper = np.argwhere(I >= I.max() / 2)
+    half = I.max() / 2
+    I_offset = I - half
```

Comment:
I do not understand why it's needed : the slit is zero on the sides.

Reply:
This steps makes interpolation easier conceptually to find the FWHM by solving the question "when do my graph crosses the y=0 axis". The function get_FWHM assumes the slit to have only 1 peak and to go to zero monotously away from the peak. You can modify it if you want, to find the y=half - but then you have to change the helper function get_zero as well (should be easy). 

I don't think I understood your comment. This function doesn't look at the sides of the slit here. It looks at the FWHM, so close to the center of the slit.

Reply:
I have just implemented the TODO XD

># TODO: Linearly interpolate at the boundary? insignificant for large number of points